Credit Card Validation/src/main/java/CreditCardValidator.java

```diff
@@ -1,13 +1,19 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+enum CardType { AMERICAN_EXPRESS }
```

Comment:
Not clear what it is:)

So, how it should look like.
You should have enum which contains payment system name and prefixes (don't forget that enum is class with specific constraints).
After it you can create or separate class like "PaymentSystemResolver" which will determine payment system by card number using  enum, or you can create method for resolving directly in the enum.